packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -167,6 +167,8 @@ class CupertinoTextField extends StatefulWidget {
     this.style,
     this.strutStyle,
     this.textAlign = TextAlign.start,
+    this.readOnly = false,
+    this.showCursor = true,
```

Comment:
I also added this functionality which I missed in the previous commit,  add test 'Cursor does not show when showCursor set to false'